README.md

```diff
@@ -0,0 +1,58 @@
+#Deploying to Heroku
+
+First make sure you have Node.js (which comes with npm as well) installed. 
+Access to the heroku and gmail acc can be found in the Slack channel.
+
+Then install the Heroku CLI (command-line interface) which also comes with a Git installer. 
+
+The app dependencies have already been installed. It is under dependencies in the package.json file. 
+**The version of Node.js that we are currently using is 6.9.5 (subject to change).** It will be used to run the app on Heroku.
+
+###Start Script
+Heroku first looks for a Procfile. If that's unavailable, then Heroku attempts to start a web process via the
+start script in **package.json**. I have currently set it up under start script because I was having some troubles
+with the Procfile. This can always change later but it is easy to see right now. 
+
+###Building your app and running it locally
+
+Install your dependencies
+```
+npm install
+```
+
+Start your app locally. You should be able to see the processes with your shell. The app should be listening on **localhost:3000**.
+
+```
+heroku local web
+```
+
+###Deploying to Heroku
+```
+git add -A 				
+git commit -m "commit message"

```

Comment:
https://devcenter.heroku.com/articles/git

Heroku says : Before you can push an app to Heroku, you’ll need to initialize a local Git repository and commit your files to it

so I think it's necessary?